src/components/ShowOnClick.js

```diff
@@ -25,10 +25,11 @@ export default hh(class ShowOnClick extends Component {
 
   render() {
     const { visible } = this.state
-    const { containerProps, button, bgProps, closeOnClick = true, closeOnEsc = true, children } = this.props
+    const { containerProps, button, disabled, bgProps, closeOnClick = true, closeOnEsc = true, children } = this.props
 
     return div(_.merge({
-        onClick: (e) => {
+        style: visible ? { zIndex: 99, position: 'relative' } : undefined,
```

Comment:
What is the significance of 99 here? Would it be worth naming this value via a constant to add clarity?

Reply:
It's purpose is purely to make sure that element is high enough, it's really a magic number. I can totally make it a constant.